docker/pravega/Dockerfile-java-11

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+FROM openjdk:11.0-jdk-slim
```

Comment:
Observing that `openjdk:11.0-jre-slim` has a smaller size(`204MB`) . We should try this docker image and update the findings.

Reply:
Changed the image to `openjdk:11.0-jre-slim`.